### PR TITLE
Edit org UI text

### PR DESCRIPTION
**Pull request by oddlittlebird**
UI text edits in the organization section of the UI.

### CODE REVIEW

**Review by dsotirakis — APPROVED** [review]
LGTM!

**Comment by oddlittlebird** [comment]
@dsotirakis, any idea why the build is failing?

**Comment by dsotirakis** [comment]
> @dsotirakis, any idea why the build is failing?

@oddlittlebird A formatting error most likely, let me check.

**Comment by dsotirakis** [comment]
@oddlittlebird Some lines exceeded the indentation line (don't know which editor you are using, so can't really tell if you have one).

I pushed some format changes, [here](https://github.com/grafana/grafana/pull/32455/files/052d09024e87c161b2b04e5eaa77b26fe5f97b43..c7d9a65fca95a30e7c0cd0f867d31e508ce2e647). Keep that in mind in case you face the same error again!

File: public/app/features/org/NewOrgPage.tsx [hunk 1]
```diff
@@ -41,12 +41,12 @@ export const NewOrgPage: FC<PropsWithState> = ({ navModel }) => {
...
       <Page.Contents>
-        <h3 className="page-sub-heading">New Organization</h3>
+        <h3 className="page-sub-heading">New organization</h3>
 
         <p className="playlist-description">
-          Each organization contains their own dashboards, data sources and configuration, and cannot be shared between
-          orgs. While users may belong to more than one, multiple organization are most frequently used in multi-tenant
-          deployments.{' '}
+          Each organization contains their own dashboards, data sources, and configuration, which cannot be shared
+          organizations. While users might belong to more than one organization, multiple organizations are most
```
Comment (natellium) [inline public/app/features/org/NewOrgPage.tsx:48]: Is the "between" removed on purpose? This reads a bit weird to me: 
> which cannot be shared organizations

Comment (oddlittlebird) [reply public/app/features/org/NewOrgPage.tsx:48]: Good catch. That was my goof.

**Review by natellium — APPROVED** [review]
Looking good! :)

**Comment by oddlittlebird** [comment]
@dsotirakis, can you see why the build is failing? I made all my new lines shorter than the lines I replaced.

**Comment by oddlittlebird** [comment]
@zuchka, might be a fun puzzle for you.

**Comment by dsotirakis** [comment]
> @dsotirakis, can you see why the build is failing? I made all my new lines shorter than the lines I replaced.

Fixed! I think they don't need to be short - you have to make sure that no other word fits in a single line before you add a new line! Confusing I know, can see that through my IDE, don't know if there is a Notepad-like tool that can provide this errors. Maybe Visual Code studio?

**Comment by oddlittlebird** [comment]
I am using VSC to edit, but it didn't give me any errors.

**Pull request merged**